frontend/src/pages/Landing/MyAlerts.js

```diff
@@ -0,0 +1,146 @@
+        className="smart-hub--table-collection"
+      >
+        {ar.grant ? ar.grant.grantee.name : ar.name}
+      </Tag>
+    ));
+
+    const collaboratorsTitle = collaborators.reduce(
+      (result, collaborator) => `${result + collaborator.fullName}\n`,
+      '',
+    );
+
+    const collaboratorsWithTags = collaborators.map((collaborator) => (
+      <Tag
+        key={collaborator.id}
+        className="smart-hub--table-collection"
+      >
+        {collaborator.fullName}
+      </Tag>
+    ));
+
+    return (
+      <tr key={`my_alerts_${id}`}>
+        <td>
+          <Link
+            to={`/activity-reports/${id}`}
+            href={`/activity-reports/${id}`}
+          >
+            {displayId}
+          </Link>
+        </td>
+        <td>
+          <span className="smart-hub--ellipsis" title={recipientsTitle}>
+            {recipients}
+          </span>
+        </td>
+        <td>{startDate}</td>
+        <td>
+          <span className="smart-hub--ellipsis" title={author.fullName}>
+            {author.fullName}
+          </span>
+        </td>
+        <td>
+          <span className="smart-hub--ellipsis" title={collaboratorsTitle}>
+            {collaboratorsWithTags}
+          </span>
+        </td>
+        <td>
+          <Tag
+            className={`smart-hub--table-tag-status smart-hub--status-${status}`}
+          >
+            {status === 'Needs_action' ? 'Needs action' : status}
+          </Tag>
+        </td>
+      </tr>
+    );
+  });
+}
+
+function MyAlerts({ reports }) {
+  return (
+    <>
+      { reports && reports.length === 0 && (
+      <Container className="landing" padding={0}>
+        <div id="caughtUp">
+          <div><h2>You&apos;re all caught up!</h2></div>
+          <p id="beginNew">Would you like to begin a new activity report?</p>
+          <NewReport />
+        </div>
+      </Container>
+      ) }
+      { reports && reports.length > 0 && (
+      <SimpleBar>
+        <Container className="landing inline-size" padding={0}>
+          <Table bordered={false}>
+            <caption className="smart-hub--table-caption">
+              My activity report alerts
+            </caption>
+            <thead>
+              <tr>
+                <th scope="col">Report ID</th>
+                <th
+                  scope="col"
+                >
+                  Grantee
+                </th>
+                <th scope="col">Start date</th>
+                <th
+                  scope="col"
+                >
+                  Creator
+                </th>
+                <th scope="col">Collaborator(s)</th>
+                <th
+                  scope="col"
+                >
+                  Status
+                </th>
+              </tr>
+            </thead>
+            <tbody>{renderReports(reports)}</tbody>
+          </Table>
+        </Container>
+      </SimpleBar>
+      )}
+    </>
+  );
+}
+
+MyAlerts.propTypes = {
+  reports: PropTypes.node.isRequired,
```

Comment:
I'm getting this in the console:

`Warning: Failed prop type: Invalid prop `reports` supplied to `MyAlerts`, expected a ReactNode`

Reply:
Changed.